widget/models.py

```diff
@@ -19,7 +20,7 @@ class WidgetHost(django.db.models.Model):
     uuid = shortuuidfield.ShortUUIDField(auto=True, unique=True, db_index=True)
     contact_email = django.db.models.EmailField()
     contact_name = django.db.models.CharField(max_length=200)
-    host_url = django.db.models.URLField(max_length=255, unique=True)
+    host_url = django.db.models.CharField(validators=[django.core.validators.URLValidator()], max_length=255, unique=True)
```

Comment:
@danaspiegel I switched it because URLField seems to be manipulating data entered by users before inserting it to the DB. So I faced a problem when comparing the URLField value with the URL came in the POST request, because a slash is always added by the URLField.

Anyways I think that by using .exist() call, this issue might be gone, so I will try to get it back to URLField and see if it works with .exist()